meeting-notes/2019-04-01--libp2p-sync.md

```diff
@@ -0,0 +1,98 @@
+# 2019-04-01 libp2p Sync Call
+
+[Issue for this call](https://github.com/libp2p/team-mgmt/issues/16)
+
+**Moderator**: @mgoelzer
+
+**Notetaker**:  @jhiesey
+
+**Recording**:  https://youtu.be/SdUN6N0Dc8o
+
+**Attendees**:
+  - @jhiesey
+  - @mgoelzer
+  - @anacrolix
+
+## Agenda
+
+- Weekly updates 
+
+## Weekly updates
+
+- @marten-seemann:
+  - Done:
+    * attented the IETF meeting in Prague
+    * quic-go has interop with most other QUIC implementations
+  - Blocked:
+    * I will merge https://github.com/libp2p/specs/pull/151 very soon, if there aren't any futher objections
+  - Next:
+    * use quic-trace to improve quic-go performance
+
+- @yusefnapora:
+  - Done:
+    - merged in a couple concept docs
+      - addressing: https://docs.libp2p.io/concepts/addressing/
+      - peer id: https://docs.libp2p.io/concepts/peer-id/
+  - Next:
+    - finish stream muxing concept doc PR: https://github.com/libp2p/docs/pull/40#event-2244416671
+    - start on protocol negotiation & encryption concept docs
+
+- @anacrolix:
+  - Done:
+    - Misc bug fixes
+    - Exploring metrics in Prometheus and OC
+  - Blocked:
+    - Meaningful results from changes to streams
+  - Next:
+    - Merge in metrics to DHT
+    - Explore tracing?
+    - Expose metrics in related repos to DHT
+    - https://github.com/libp2p/go-libp2p-kad-dht/issues/295
+    - https://github.com/libp2p/go-libp2p-kad-dht/issues/283
+
+- @jhiesey:
+  - Done:
+    - Met with @jbenet about DHT design
+      - He liked my proposal (https://gist.github.com/jhiesey/f9a3d9961ccdfca940e54ec279b47cf9) generally
+      - Agreed that we need to modularize things
+      - Some parts of current DHT belong in other parts of libp2p
+      - Got some new papers to read
+  - Blocked:
+    - none
+  - Next:
+    - Read @jbenet's suggested papers:
```

Comment:
We do usually record journal clubs but, as you said, it's not always feasible to deep link to specific exchanges. Nevertheless, I think it could be very valuable for those in attendance as it's much more conducive to discussion than just posting summaries on github. Someone could then write a summary md with a link to the full recording for those who wanted to dig deeper.

You could also use Mendeley to collaboratively annotate papers, but that would require everyone creating an account. It would only make sense if you were to do this regularly.

Reply:
@raulk Have you considered open discussion in the https://discuss.libp2p.io/c/research?  Just a thought.

Reply:
Oh, dear. Yeah, that's the right place. What an oversight 🤦‍♂️